quiz.py

```diff
@@ -1,6 +1,64 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Matthew Daguanno
+Date: 2024-03-04
+Description: Asks 3 questions and gives 3 options to pick from for each.
+"""
+
+print("Multiple Choice Quiz Game")
+
+counter = 0
+
+# Question one
+q1_inp = input("1. What is the capital of France?")
+print("(a) Paris")
+print("(b) London")
+print("(c) Rome")
```

Comment:
Have you tested this? Your functionality doesn't reveal the choices until **after** the user has given some sort of input. That doesn't seem right.